utility/ln_config.h

```diff
@@ -55,6 +55,21 @@
  // figure out what board we are building
 
  // Common defines
+ 
+// turn off EEPROM for platforms not currenty implemented
```

Comment:
Why are you doing a #define LOCONET_NO_EEPROM by default for each of these platforms when they have a EEPROM emulation in FLASH already?

Reply:
because EEPROM support is only implemented for STM32F1, ESP8266 and AVR.
EEPROM is by default off for all other platforms, and may additionally turned off for ESP8266, STM and AVR